services/frontend/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -9,6 +9,21 @@ const handler = NextAuth({
     }),
   ],
   callbacks: {
+    async jwt({ token, user }) {
+      try {
+        if (user) {
+          const res = await fetch(
+            `http://${process.env.NEXT_PUBLIC_SERVICE_USER_URL}/user?email=${user.email}`,
+          );
+          const registeredUser = await res.json();
```

Comment:
There could be an instance where `fetch` deoes not throw an error but `res.ok` is false. We should account for this case. We can throw an error here, and let the error handling happen in the catch block, where we dispatch a notification ya

```suggestion
          const res = await fetch(
            `http://${process.env.NEXT_PUBLIC_SERVICE_USER_URL}/user?email=${user.email}`,
          );
          if (!res.ok) {
            throw new Error('HTTP error! status: ${res.status}');
          }
          const registeredUser = await res.json();
```